lib/tasks/docker.rake

```diff
@@ -85,18 +90,20 @@ task 'docker:test' do
 
     unless ENV["RUBY_ONLY"]
       unless ENV["SKIP_CORE"]
-        @good &&= run_or_fail("eslint app/assets/javascripts")
-        @good &&= run_or_fail("eslint --ext .es6 app/assets/javascripts")
-        @good &&= run_or_fail("eslint --ext .es6 test/javascripts")
-        @good &&= run_or_fail("eslint test/javascripts")
+        @good &&= run_or_fail("eslint app/assets/javascripts") unless ENV["SKIP_LINT"]
+        @good &&= run_or_fail("eslint --ext .es6 app/assets/javascripts") unless ENV["SKIP_LINT"]
+        @good &&= run_or_fail("eslint --ext .es6 test/javascripts") unless ENV["SKIP_LINT"]
+        @good &&= run_or_fail("eslint test/javascripts") unless ENV["SKIP_LINT"]
```

Comment:
Lets do it in a single command `eslint app/assets/javascripts/ test/javascripts/`